packages/scene-composer/src/components/panels/scene-components/AnimationComponentEditor.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useCallback, useContext } from 'react';
+
+import { useStore, ISceneNodeInternal } from '../../../store';
+import { sceneComposerIdContext } from '../../../common/sceneComposerIdContext';
+
+import { AnimationViewStateEditor, AnimationViewStateEditorProps } from './AnimationViewStateEditor';
+
+export interface AnimationComponentEditorProps extends AnimationViewStateEditorProps {
+  node: ISceneNodeInternal;
+  component: any;
```

Comment:
looks like they are the same as the base type, so you don't need to define them here again